lib/logger.js

```diff
@@ -8,10 +8,19 @@ let loggerTransports = [
     level: logLevel,
     filename: './log/discovery-api.log',
     handleExceptions: true,
-    json: true,
     maxsize: 5242880, // 5MB
     maxFiles: 5,
-    colorize: false
+    colorize: false,
+    json: false,
```

Comment:
The `formatter` function will return a JSON string but you HAVE to turn `json: false` here for the formatter to even be evaluated. [See the `formatter` option here.](https://github.com/winstonjs/winston/blob/master/docs/transports.md#file-transport)